go/libkb/paperkey_phrase.go

```diff
@@ -34,7 +34,8 @@ func MakePaperKeyPhrase(version uint8) (PaperKeyPhrase, error) {
 
 // NewPaperKeyPhrase converts a string into a PaperKeyPhrase.
 func NewPaperKeyPhrase(phrase string) PaperKeyPhrase {
-	return PaperKeyPhrase(strings.ToLower(phrase))
+	s := strings.Join(strings.Fields(phrase), " ")
```

Comment:
@patrickxb did you see my second comment above?  (You replied to only the first)